src/views/Profile/PersonalInfo/PersonalInfo.tsx

```diff
@@ -201,9 +227,12 @@ export const PersonalInfo: React.FC<IEditAuthorProps> = ({
         if (errorMessages[key]) {
           return true;
         }
+        if (isLoading) {
```

Comment:
I think `isLoading` should be checked for `true` outside of `Object.keys(errorMessages)`. 
maybe even with separate const:

```
const anyFieldWithError = useMemo(....);
const isSaveDisabled = anyFieldWithError || isLoading;
```

It's pretty minor in effect, but I believe it gives more clarity

Reply:
Okay, i will refactor this